addons/route53-mapper/v1.3.0.yml

```
@@ -1,5 +1,5 @@
 ---
-apiVersion: extensions/v1beta1
+apiVersion: app/v1
```

```suggestion
apiVersion: apps/v1
```
I believe it should be plural

See https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.16/#deployment-v1-apps

---

Transcription error. My apologies.